linera-sdk/src/bin/linera-wasm-test-runner/mock_system_api.rs

```diff
@@ -928,6 +928,94 @@ pub fn add_to_linker(linker: &mut Linker<Resources>) -> Result<()> {
         },
     )?;
 
+    linker.func_wrap2_async(
+        "view_system_api",
+        "read-multi-values-bytes::new: \
+            func(key: list<list<u8>>) -> handle<read-multi-values-bytes>",
```

Comment:
Nit: if the `linera-sdk/view_system_api.wit` is updated to use `keys` instead of `key`, this should be updated as well.

```suggestion
        "read-multi-values-bytes::new: \
            func(keys: list<list<u8>>) -> handle<read-multi-values-bytes>",
```